app/code/Magento/InventoryConfigurableProduct/view/adminhtml/web/js/variations/steps/summary.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'Magento_ConfigurableProduct/js/variations/steps/summary',
+    'jquery',
+    'mage/translate'
+], function (Summary, $) {
+    'use strict';
+
+    return Summary.extend({
+        /** @inheritdoc */
+        initObservable: function () {
+            this._super();
+            this.quantityFieldName = 'quantityPerSource';
```

Comment:
Please move configuration to `defaults` block. Also, you use `initObserveble` method don't by purpose. In `initObserveble` you should declare observable variables and arrays, in other cases, you should use `initialize` or `initConfig` methods.

Reply:
Ok. Fixed.